index.html

```diff
@@ -1,22 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
-  <title>Awesome Books</title>
-</head>
-<body>
-  <h1>Awesome Library</h1>
-  <ul id="bookList">  
-  </ul>
-  <form action="#">
-    <input type="text" placeholder="Title" id="title"><br>
-    <input type="text" placeholder="Author" id="author"><br><br>
-    <button type="button" onclick="lib.addBook()">Add New Book</button> 
-  </form>
-  <p id ="test"></p>
-  <script src="script.js"></script>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Zen+Loop&display=swap"
+      rel="stylesheet"
+    />
+    <title>Awesome Books</title>
+  </head>
+  <body class="container text-center">
+    <div class="m-4">
+      <h1 class="display-3">Awesome Library</h1>
+    </div>
+    <div class="border border-3 rounded w-50 m-auto">
+      <ul id="bookList" class="list-unstyled"></ul>
+    </div>
+    <div class="mt-4 w-25 m-auto">
+      <h1 class="display-5 pt-3 border-top border-3">Add a new book</h1>
+      <form action="#">
+        <input
+          class="w-100 p-1 mt-2 border-2 rounded"
+          type="text"
+          placeholder="Title"
+          id="title"
+        /><br />
+        <input
+          class="w-100 p-1 mt-2 border-2 rounded"
+          type="text"
+          placeholder="Author"
+          id="author"
+        /><br />
+        <button
+          class="float-end mt-2 border-2 rounded"
+          type="button"
+          onclick="lib.addBook()"
+        >
+          Add New Book
+        </button>
+      </form>
```

Comment:
#[Optional]
Good job guys..... 👍 
kindly validate your form, a user should not be able to add an empty form.
In another word; You need to do form controls to make sure empty forms cannot be submitted.
Screenshot from your app;
![image](https://user-images.githubusercontent.com/31897434/124912139-7fd36f80-dfe5-11eb-9efe-4a275187af82.png)


Hint: You could apply `required` to each `input` tag.